### PR TITLE
Added go-recipes badge

### PR DESCRIPTION
**Pull request by nikolaydubina**
Hello, Hi!! 👋🏻

I like your project and I think broader Go community will benefit from it too. 

Thus, I added it to the curated list of Go tools.

I hope this badge will serve as a mark of quality and appreciation to your project.

Once again, thank you for your work!!

❤️

-- Nikolay

### CODE REVIEW

**Review by bobheadxi — COMMENTED** [review]
hey Nikolay, thanks for including gobenchdata on your list - happy to accept this PR, but would appreciate if you could move the badge to the very end of the list of badges :) thanks!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # 📉 gobenchdata
 
 [![View Action](https://img.shields.io/badge/view-github%20action-yellow.svg)](https://bobheadxi.dev/r/gobenchdata)
+[![go-recipes](https://raw.githubusercontent.com/nikolaydubina/go-recipes/main/badge.svg?raw=true)](https://github.com/nikolaydubina/go-recipes)
 [![publish@v1](https://github.com/bobheadxi/gobenchdata/workflows/publish@v1/badge.svg)](https://github.com/bobheadxi/gobenchdata/actions?workflow=publish%40v1)
 [![pipeline](https://github.com/bobheadxi/gobenchdata/workflows/pipeline/badge.svg)](https://github.com/bobheadxi/gobenchdata/actions?workflow=pipeline)
 [![demo](https://github.com/bobheadxi/gobenchdata/workflows/demo/badge.svg)](https://github.com/bobheadxi/gobenchdata/actions?workflow=demo)
```
Comment (nikolaydubina) [inline README.md:7]:
```suggestion
[![View Action](https://img.shields.io/badge/view-github%20action-yellow.svg)](https://bobheadxi.dev/r/gobenchdata)
[![publish@v1](https://github.com/bobheadxi/gobenchdata/workflows/publish@v1/badge.svg)](https://github.com/bobheadxi/gobenchdata/actions?workflow=publish%40v1)
[![pipeline](https://github.com/bobheadxi/gobenchdata/workflows/pipeline/badge.svg)](https://github.com/bobheadxi/gobenchdata/actions?workflow=pipeline)
[![demo](https://github.com/bobheadxi/gobenchdata/workflows/demo/badge.svg)](https://github.com/bobheadxi/gobenchdata/actions?workflow=demo)
[![go-recipes](https://raw.githubusercontent.com/nikolaydubina/go-recipes/main/badge.svg?raw=true)](https://github.com/nikolaydubina/go-recipes)
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # 📉 gobenchdata
 
 [![View Action](https://img.shields.io/badge/view-github%20action-yellow.svg)](https://bobheadxi.dev/r/gobenchdata)
+[![go-recipes](https://raw.githubusercontent.com/nikolaydubina/go-recipes/main/badge.svg?raw=true)](https://github.com/nikolaydubina/go-recipes)
 [![publish@v1](https://github.com/bobheadxi/gobenchdata/workflows/publish@v1/badge.svg)](https://github.com/bobheadxi/gobenchdata/actions?workflow=publish%40v1)
```
Comment (nikolaydubina) [inline README.md:5]:
```suggestion
[![publish@v1](https://github.com/bobheadxi/gobenchdata/workflows/publish@v1/badge.svg)](https://github.com/bobheadxi/gobenchdata/actions?workflow=publish%40v1)
[![go-recipes](https://raw.githubusercontent.com/nikolaydubina/go-recipes/main/badge.svg?raw=true)](https://github.com/nikolaydubina/go-recipes)
```

**Comment by nikolaydubina** [comment]
@bobheadxi updated! moved to new PR! https://github.com/bobheadxi/gobenchdata/pull/67

**Pull request closed without merging**